test/e2e/load.go

```diff
@@ -139,18 +154,18 @@ func computeRCCounts(total int) (int, int, int) {
 	return smallRCCount, mediumRCCount, bigRCCount
 }
 
-func generateRCConfigs(totalPods int, c *client.Client, ns string) []*RCConfig {
+func generateRCConfigs(totalPods int, image string, command []string, c *client.Client, ns string) []*RCConfig {
```

Comment:
Why not? If we have 2 node cluster (like in e2e) there will be 1 RC with 5 pods. If we have 100 node custer (scalability) there will be 3 RC with 30 pods and ~ 40 RC with 5 pods. It will also simulate load on Kubernetes components.